shared/lib/package.json

```diff
@@ -18,5 +18,8 @@
     "js-yaml": "^4.1.0",
     "minimatch": "^9.0.3",
     "slash": "^3.0.0"
+  },
+  "devDependencies": {
+    "@types/js-yaml": "^4.0.5"
```

Comment:
When adding `paths` to `tsconfig.base.json`, this gets flagged

Reply:
Ahh probably because we have strict mode turned on?

Would you mind adding it to the root of the project inside `optionalDependencies`?

Keeps all our optional types together